lib/rspec/matchers/built_in/respond_to.rb

```diff
@@ -130,6 +132,12 @@ def matches_arity?(actual, name)
           expectation.expect_unlimited_arguments = @unlimited_arguments
           expectation.expect_arbitrary_keywords  = @arbitrary_keywords
 
```

Comment:
Unrelated. Just wondering how strict is it to only consider numeric `@expected_arity` as a `min`.
Can it be:
```ruby
expectation.min_count = @expected_arity
expectation.max_count = @expected_arity
```

Reply:
Its set to min later on in rspec-support if its never set, its a bit more complex than doing this.